wrangler/src/trust/heal.js

```diff
@@ -0,0 +1,15 @@
+import { response } from 'cfw-easy-utils'
+import { heal } from 'trust.js'
+
+export default async ({ request, env }) => {
+  const { TX_FUNCTIONS, TURRET_ADDRESS, STELLAR_NETWORK, HORIZON_URL } = env
+  const { 
+    sourceaccount,
+    oldturret,
+    newturret,
+    functionhash 
+  } = await request.json()
+  
+  healresponse = await heal(sourceaccount, oldturret, newturret, functionhash)
```

Comment:
unsure why the compiler let you avoid placing a `let` or `const` here.

Add one for consistency?